### ISSUE TITLE
Different COF's between different objects

### ISSUE THREAD

**Issue by Untuxed** [polyfem/polyfem#198]
**Describe the solution you'd like**
Assign different COFs per interaction.  

**Additional context**
Assign a different COF based on volume IDs. The default should still be a global COF.

**Comment by zfergus** [comment]
What are COFs?

**Comment by zfergus** [comment]
Oh, coefficient of friction?

We have discussed this but one of the theoretical blockers was how to best blend different coefficients of friction. 

One option is to define one per pair of surfaces (similar to how they are listed on [Wikipedia](https://en.wikipedia.org/wiki/Friction#Approximate_coefficients_of_friction)). Do you have any ideas on this? Is there one you would like based on another standard software or common implementation?

**Comment by li4mm4rtin** [comment]
Yeah coefficients of friction, sorry I forgot to respond to this. I will look into methods of blending COFs and report back on Monday. There isn’t a common implementation, just a shortcoming of most commercial solvers that would be nice to address.

**Comment by antoinebou12** [comment]
If it is not implemented here, i going to give my proposal

[ETSTribology/polyfem.github.io@friction/docs/tutorials/frictions/frictions.md](https://github.com/ETSTribology/polyfem.github.io/blob/friction/docs/tutorials/frictions/frictions.md?rgh-link-date=2024-10-23T23%3A49%3A19Z)
[antoinebou12/polyfem@friction](https://github.com/antoinebou12/polyfem/tree/friction?rgh-link-date=2024-10-23T23%3A49%3A19Z)
